cachito/web/content_manifest.py

```diff
@@ -91,7 +91,6 @@ def process_gomod(self, package, dependency, type="icm"):
         :param type: icm or sbom component
         """
         if dependency.type == "gomod":
-
```

Comment:
Are these changes caused by some automatic linter you're using in the IDE? I'd say it's probably better to avoid reformating files we're not actually changing in this PR.